### PR TITLE
Nurse PDA, Skirt of Life, Alt Nurse Hat

### PR DESCRIPTION
**Pull request by PursuitInAshes**
## About the PR
Revives #109 .

Adds the Skirt of Life and a recolored Nurse Hat to the Medical Doctor loadout options.
Adds a new PDA, the Nurse PDA, which uses CD's job title system.

## Why / Balance
More RP options for medical staff.

## Technical details
N/A - YAML

## TODO
Make the PDA Icon load

### CODE REVIEW

**Comment by PursuitInAshes** [comment]
So it turns out putting the sprite in the upstream folder (and adjusting the yaml), makes the sprite load just fine. I'll take a look at the file path again because thats my best guess.

**Comment by PursuitInAshes** [comment]
I'll come back to fix this later, the changes are done, I just have to rebase properly instead of whatever I did.

**Comment by PursuitInAshes** [comment]
Just confirmed that it still works ingame, this is ready for review.

File: Resources/Prototypes/Loadouts/loadout_groups.yml [hunk 1]
```diff
@@ -1210,6 +1210,7 @@
   - GreenSurgeryCap
   - PurpleSurgeryCap
   - NurseHat
+  - NurseHatAlt #Umbra
```
Comment (Coolsurf6) [inline Resources/Prototypes/Loadouts/loadout_groups.yml:1213]: Just a space needed to make # Umbra to remain consistent.

**Review by Coolsurf6 — CHANGES_REQUESTED** [review]
Everything aside from a comment seems fine and runs in game well.

**Review by Coolsurf6 — APPROVED** [review]
Change is done, looks good.

**Pull request merged**